views/explorer.tmpl

```diff
@@ -122,8 +109,8 @@
                         <td class="text-center" data-type="value">{{threeSigFigs .Total}}</td>
                         <td class="text-center" data-type="size">{{.FormattedBytes}}</td>
                         <td class="text-center d-none d-sm-table-cell" data-type="version">{{.Version}}</td>
-                        <td class="text-right" data-type="age" class="jsonly" data-target="time.age" data-age="{{.BlockTime.UNIX}}"></td>
-                        <td class="text-right px-0" data-type="time">{{.BlockTime}}</td>
+                        <td class="text-center" data-type="age" class="jsonly" data-target="time.age" data-age="{{.BlockTime.UNIX}}"></td>
+                        <td class="text-center px-0" data-type="time">{{.BlockTime}}</td>
```

Comment:
See note above. This needs to be right-aligned, as per the mockup and general table rules, but it's difficult to get the header centered over the row.